docs/_docs/configuration.md

```diff
@@ -13,6 +13,38 @@ title: Configuration
 {:toc}
 
 ---
+## Configuration
+
+Rhino supports reading configuration from configuration files in the upcomming release (> 1.8.0)
+
+By default, rhino reads the configuration from
+-  `rhino.config` file from current class' classpath
+-  `rhino.config` file from current threas's classpath
+-  `rhino.config` file from current directory
+-  `rhino-test.config` file from current class' classpath
+-  `rhino-test.config` file from current threas's classpath
+-  `rhino-test.config` file from current directory
+- env variables (normally starting with "RHINO_")
+- System-properties (normally starting with "rhino.")
+
+Properties can be specified in camel case dot notation (e.g. `rhino.printICode=true`) or in upper case underscore notation (e.g. `RHINO_PRINT_ICODE=true`) 
+
+Currently available flags:
+- `rhino.stack.style` default value `rhino`. Available `mozilla`, `mozilla_lf`, `v8`
+- `rhino.printTrees` default value `false`
+- `rhino.printICode` default value `false`
+- `rhino.debugLinker` default value `false`
+- `rhino.use_java_policy_security` default value `false`
+
+### Notes for contributors
```

Comment:
This section for contributors might be more useful in the main repo under Contributing